hcx-apis/src/main/resources/application.yml

```diff
@@ -0,0 +1,23 @@
+#kafka configuration
+bootstrap-servers: ${bootstrap_servers:localhost:9092}
+
+kafka:
+  topic:
+    payload: ${kafka_payload_topic:local.hcx.request.payload}
+    ingest: ${kafka_ingest_topic:local.hcx.request.ingest}
+
+#hcx headers
+protocol:
+  mandatory:
+    headers: ${protocol_mandatory_headers:x-hcx-sender_code, x-hcx-recipient_code, x-hcx-request_id, x-hcx-correlation_id, x-hcx-timestamp, x-hcx-status, jws_header, jwe_header}
```

Comment:
```suggestion
protocol:
  headers:
    mandatory: ${protocol_mandatory_headers:x-hcx-sender_code, x-hcx-recipient_code, x-hcx-request_id, x-hcx-correlation_id, x-hcx-timestamp, x-hcx-status, jws_header, jwe_header}
```

Reply:
Please apply the same change for all the other header configurations.

Reply:
Done.